docs/fundamentals/code-analysis/quality-rules/ca1024.md

```diff
@@ -25,41 +25,35 @@ dev_langs:
 
 ## Rule description
 
 In most cases, properties represent data and methods perform actions. Properties are accessed like fields, which makes them easier to use. A method is a good candidate to become a property if one of these conditions is present:
 
-- Takes no arguments and returns the state information of an object.
+- The method takes no arguments and returns the state information of an object.
+- The method accepts a single argument to set some part of the state of an object.
 
-- Accepts a single argument to set some part of the state of an object.
-
-Properties should behave as if they are fields; if the method cannot, it should not be changed to a property. Methods are better than properties in the following situations:
+## How to fix violations
 
-- The method performs a time-consuming operation. The method is perceivably slower than the time that is required to set or get the value of a field.
+To fix a violation of this rule, change the method to a property.
 
-- The method performs a conversion. Accessing a field does not return a converted version of the data that it stores.
+## When to suppress warnings
 
-- The Get method has an observable side effect. Retrieving the value of a field does not produce any side effects.
+Suppress a warning from this rule if the method meets one of the following criteria. In these situations, a method is preferable to a property.
 
+- The method can't behave as a field.
+- The method performs a time-consuming operation. The method is perceivably slower than the time that is required to set or get the value of a field.
+- The method performs a conversion. Accessing a field does not return a converted version of the data that it stores.
+- The `Get` method has an observable side effect. Retrieving the value of a field does not produce any side effects.
 - The order of execution is important. Setting the value of a field does not rely on the occurrence of other operations.
-
 - Calling the method two times in succession creates different results.
-
-- The method is static but returns an object that can be changed by the caller. Retrieving the value of a field does not allow the caller to change the data that is stored by the field.
-
+- The method is `static` but returns an object that can be changed by the caller. Retrieving the value of a field does not allow the caller to change the data that's stored by the field.
 - The method returns an array.
 
-## How to fix violations
-
-To fix a violation of this rule, change the method to a property.
-
-## When to suppress warnings
-
-Suppress a warning from this rule if the method meets at least one of the previously listed criteria.
+In addition, if your class implements the <xref:System.Xml.Serialization.IXmlSerializable> interface, it's okay to suppress the CA1024 warning on your <xref:System.Xml.Serialization.IXmlSerializable.GetSchema> method implementation.
```

Comment:
This is the "new" suppression info. The rest was copy/pasted from the Description section.

Reply:
I believe the rule should not fire on interface implementations - this feels like a potential bug.

Reply:
I've asked the OP to log a bug.